### DISCUSSION TITLE
This is a work in progress. Use at your own risk

### DISCUSSION THREAD

**Discussion by DusanLesan** [gokcehan/lf#1395] [General]
Are devs still thinking this?
From my perspective, you have moved well past the stage of just a minimal required implementation, with numerous nice-to-have features implemented. Considering this and the fact that all recent potentially braking changes have been approached with great care, it seems fitting to remove this notice as it could give wrong impression of the project

**Comment by joelim-work** [comment]
My opinion is that `lf` is somewhere in the middle in terms of maturity. It isn't fully mature (and perhaps never will be) as new features are still being worked on, and existing features are being refined causing releases to have breaking changes. But it also past the infancy stage, considering the number of users, and the fact that a lot of care is taken to minimize impact to users when introducing breaking changes.

Regarding the 'work in progress' notice, I have never really thought about it before, but I guess I am in favor of removing it. You are more than welcome to submit a PR to update the README so that the project will look more attractive to potential new users.

**Reply by DusanLesan** [reply]
I have no ideas other than simply removing that line. I think it should be reserved for projects that are still in its beta stage.
It is completely normal that active project introduces new features and sometimes break things. The new features included lately are great but are mostly non-essential. I do not know if this project have any unfulfilled main goals, and that leads me to think it is fully mature as opposed to completely reaching end of development

**Reply by gokcehan** [reply]
@DusanLesan To be honest, I forgot we still have that in the readme. I agree it does not make much sense at this point, so I have now removed it. I had added that notice when I started the project since file managers are inherently dangerous as they involve file operations. We still experience such issues from time to time (e.g. #1314). However, I don't think there will ever come a point that we can guarantee 100% safety in a file manager. So I think such a risk should be implicitly assumed when using a file manager. This is also somehow already mentioned in open-source licenses in general.
